apps/journeys-admin/src/components/Editor/Toolbar/Items/AnalyticsItem/AnalyticsItem.tsx

```diff
@@ -15,17 +16,19 @@ export function AnalyticsItem({ variant }: AnalyticsItemProps): ReactElement {
   const { t } = useTranslation('apps-journeys-admin')
   const { journey } = useJourney()
   return (
-    <NextLink
-      href={`/journeys/${journey?.id}/reports`}
-      passHref
-      legacyBehavior
-      prefetch={false}
-    >
-      <Item
-        variant={variant}
-        label={t('Analytics')}
-        icon={<BarChartSquare3Icon />}
-      />
-    </NextLink>
+    <Box data-testId="Analytics-Item">
```

Comment:
Use PascalCase for the test id. Please also note the correct casing on `data-testid`.
Make this change for other relevant places too
```suggestion
    <Box data-testid="AnalyticsItem">
```